### DISCUSSION TITLE
Modify template (Create new version) via AWS CLI

### DISCUSSION THREAD

**Discussion by ShafrirK** [aws/aws-cli#8005] [Q&A] [answered]
Hello,

I'm trying to Modify template (Create new version) from AWS CLI via Powershell with this code:

aws ec2 create-launch-template-version --launch-template-id lt-3289dh298dh2389 --version-description WebVersion2 --source-version 1 --launch-template-data '{"ImageId":"ami-0293d939101d983ab"}'

I receive this error:
Error parsing parameter '--launch-template-data': Invalid JSON: Expecting property name enclosed in double quotes: line 1 column 2 (char 1)
JSON received: {ImageId:ami-0293d939101d983ab}

the code syntax was taken directly from AWS CLI create-launch-template-version page (image attached).

does anyone knows what I can do to solve this issue?

Thanks

![Untitled](https://github.com/aws/aws-cli/assets/137901552/18e8b25c-035b-464f-80b8-217d67c3a83c)

**Comment by tim-finnigan** [comment, marked as answer]
Found a related issue: https://github.com/aws/aws-cli/issues/6035. As mentioned there this appears to be expected behavior and a limitation of Powershell. Can you try escaping the double quotes as documented here: https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-parameters-quoting-strings.html#powershell
